content/common/linux.mk

```
@@ -0,0 +1,12 @@
+CC := gcc
+CFLAGS ?= -g -Wall -Wextra -Werror -I ../utils
```

```suggestion
CPPFLAGS += -I ../utils
CFLAGS += -g -Wall -Wextra -Werror
```
I would append these: `+=`

---

I would create a separate file `defs.mk` where I would add these variables.

This is for the case where I would use custom rules, custom dependencies and including `linux.mk` or `single.mk` / `multiple.mk` is too much.